client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -19,157 +20,78 @@ import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import Card from 'components/card';
 import Button from 'components/button';
 import SectionHeader from 'components/section-header';
-import { protectForm } from 'lib/protect-form';
+import { isJetpackSite, isJetpackModuleActive } from 'state/sites/selectors';
 
-const debug = debugFactory( 'calypso:my-sites:site-settings' );
-
-const SiteSettingsFormDiscussion = protectForm( React.createClass( {
-	mixins: [ formBase ],
-
-	discussionAttributes: [
-		'default_pingback_flag',
-		'default_ping_status',
-		'default_comment_status',
-		'require_name_email',
-		'comment_registration',
-		'close_comments_for_old_posts',
-		'close_comments_days_old',
-		'thread_comments',
-		'thread_comments_depth',
-		'page_comments',
-		'comments_per_page',
-		'default_comments_page',
-		'comment_order',
-		'comments_notify',
-		'moderation_notify',
-		'social_notifications_like',
-		'social_notifications_reblog',
-		'social_notifications_subscribe',
-		'comment_moderation',
-		'comment_whitelist',
-		'comment_max_links',
-		'moderation_keys',
-		'blacklist_keys',
-		'admin_url',
-		'wpcom_publish_comments_with_markdown',
-		'markdown_supported',
-	],
-
-	getSettingsFromSite( siteInstance ) {
-		const site = siteInstance || this.props.site,
-			settings = {};
-
-		if ( site.settings ) {
-			this.discussionAttributes.map( attribute => {
-				settings[ attribute ] = site.settings[ attribute ];
-			} );
-		}
-
-		settings.fetchingSettings = site.fetchingSettings;
-
-		return settings;
-	},
-
-	resetState() {
-		const clearSettings = { fetchingSettings: true };
-		this.discussionAttributes.forEach( attribute => {
-			clearSettings[ attribute ] = '';
+class SiteSettingsFormDiscussion extends Component {
+	handleCommentOrder = () => {
+		this.props.trackEvent( 'Comment Order on Page' );
```

Comment:
I think this was missed in the last set of changes to return to 'Toggled Comment Order on Page'